srcpkgs/VVVVVV/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'VVVVVV'
+pkgname=VVVVVV
+version=2.3.6
+revision=1
+build_wrksrc=desktop_version
+build_style=cmake
+configure_args="$(vopt_bool makeandplay MAKEANDPLAY)"
+makedepends="SDL2-devel SDL2_mixer-devel"
+short_desc="Platformer exploring what if you reversed gravity instead of jumping?"
+maintainer="Luca Matei Pintilie <luca@lucamatei.com>"
+license="custom:Proprietary"
+homepage="https://thelettervsixtim.es/"
+changelog="https://github.com/TerryCavanagh/VVVVVV/releases"
+distfiles="https://github.com/TerryCavanagh/VVVVVV/archive/refs/tags/$version.tar.gz
+https://thelettervsixtim.es/makeandplay/data.zip"
+checksum="a3366aab9e8462d330044ab1ec63927e9f5c3801c0ed96b24f08c553dcb911e9
+ c767809594f6472da9f56136e76657e38640d584164a46112250ac6293ecc0ea"
+repository=nonfree
+skip_extraction="data.zip"
+
+build_options="makeandplay"
+desc_option_makeandplay="Disable official levels"
+build_options_default="makeandplay"
+
+do_install() {
+	vbin build/VVVVVV
+	vinstall "$XBPS_SRCDISTDIR/$pkgname-$version/data.zip" 644 /var/lib/vvvvvv/
+}
+
+post_install() {
+	vlicense ../LICENSE.md
```

Comment:
```suggestion
	vlicense ../LICENSE.md
	vlicense "../License exceptions.md"
```
That second document contains the exception that permits distribution of the Make and Play edition